akka-persistence/src/main/scala/akka/persistence/journal/ReplayFilter.scala

```diff
@@ -46,6 +53,10 @@ private[akka] class ReplayFilter(persistentActor: ActorRef, mode: ReplayFilter.M
   import JournalProtocol._
   import ReplayFilter.{ Warn, Fail, RepairByDiscardOld, Disabled }
 
+  // for binary compatibility
+  def this(persistentActor: ActorRef, mode: ReplayFilter.Mode,
+           windowSize: Int, maxOldWriters: Int) = this(persistentActor, mode, windowSize, maxOldWriters, debugEnabled = false)
+
```

Comment:
None of the changes in this file are really needed since it's internal api, but Roland preferred to make them binary backwards compatible anyway and then all of it should be changed.